runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -1047,6 +1047,18 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 			continue;
 		}
 
+#if defined(J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION)
+		if (try_scan(&scan_start, "enableVirtualLargeObjectHeap")) {
+			extensions->isVirtualLargeObjectHeapRequested = true;
+			continue;
+		}
+
+		if (try_scan(&scan_start, "disableVirtualLargeObjectHeap")) {
+			extensions->isVirtualLargeObjectHeapRequested = false;
+			continue;
+		}
+#endif /* defined(J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION) */
```

Comment:
My suggestion is to remove ifdef wrapping here. Field in extensions is created unconditionally and exists already. These options are going to set field unconditionally regardless is it used or not. So the choice is should we prevent JVM to start if option is provided but feature is not fully supported or not. I even think this piece can be extracted from this PR and merged separately right now. @amicic opinion?